### ISSUE TITLE
Update to latest Google maps version

### ISSUE THREAD

**Issue by bart-1990** [Carrooi/Js-GoogleMapsLoader#58]
Is it possible to update to the latest Google Maps version, '3.31'? I think all you need to do is change line 21 to `var googleVersion = '3.31';`

Also, is it possible to use a new version after this commit, so we can easily update this package? Thank you in advance!

**Comment by TheBearSO** [comment]
That's the default version ([See line 53](https://github.com/Carrooi/Js-GoogleMapsLoader/blob/develop/lib/Google.js#L53)), so you can override it simply doing the following:

```
import GoogleMapsLoader from 'google-maps';
GoogleMapsLoader.VERSION = '3.31';
```

Check the available options to override on docs: https://github.com/Carrooi/Js-GoogleMapsLoader#options

**Comment by bart-1990** [comment]
Yes, that did the trick. Thank you!

**Comment by mt404** [comment]
Google released a new version of their JS api yesterday.  Am trying to peg our application back to the previous version.  Unfortunately the following doesn't seem to have any effect:

`googleMapsLoader.VERSION = '3.30';`

Still ending up with: google.maps.version "3.32.1a"

Any ideas on how to troubleshoot?

**Comment by mt404** [comment]
Figured it out.  the VERSION only gets appended if a CLIENT is specified. 

Is there a reason we can't set a version without that?

**Comment by TheBearSO** [comment]
Hi @mt404, about your [first comment](https://github.com/Carrooi/Js-GoogleMapsLoader/issues/58#issuecomment-365755035) are you using webpack or similar modular tool? (just to know more about it)

<hr>

And about your[ second comment](https://github.com/Carrooi/Js-GoogleMapsLoader/issues/58#issuecomment-365758342), you can see why this happened [#109](https://github.com/Carrooi/Js-GoogleMapsLoader/blob/develop/lib/Google.js#L109) I don't understand why you should set the CLIENT to set the VERSION. Maybe the creator know more about it, and I'm pretty sure that this line has a reason. If you ask to me, this should be like:

```
GoogleMapsLoader.createUrl = function() {
    //...
    if (GoogleMapsLoader.VERSION) {
	url += '&v=' + GoogleMapsLoader.VERSION;
    }
};
```

**Comment by mt404** [comment]
@TheBearSO - ya, we're using webpack to build our vue.js app.  

Have worked around the issue by setting .CLIENT to the same value as .KEY.  This doesn't appear to cause any unintended consequences at the moment.  But don't think it's being used as intended.

**Comment by BrettHunter** [comment]
Setting .CLIENT to same value as .KEY results in a Google `KeyLooksLikeClientId` warning and caused the places API to disregard the .KEY value.  I hacked around this issue like so:
```
GoogleMapsLoader.VERSION = "3.31";
GoogleMapsLoader.LIBRARIES = ["geometry", "places"];
const GoogleApiKey = "API_KEY";
GoogleMapsLoader.KEY = `${GoogleApiKey}&v=${GoogleMapsLoader.VERSION}`;
```

**Comment by zxc23** [comment]
I got caught out by 3.32 badly. When I went to specify version 3.31, I ran into the above issue.

TheBearSO's fix above should be committed.

**Comment by davidkudera** [comment]
Hi, @TheBearSO is correct. The original condition should be split into two. Does anyone want to submit a pull request?

**Comment by TheBearSO** [comment]
@davidkudera I will do it. Now I'm working, but when I get home I can add that line.

**Comment by davidkudera** [comment]
The google api version with configuration fix was just released in version 3.3.0.

Thanks again @TheBearSO :beers:
